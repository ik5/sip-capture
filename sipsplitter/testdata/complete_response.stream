SIP/2.0 200 OK
Content-Length: 0

