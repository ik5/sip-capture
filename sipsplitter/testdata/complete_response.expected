{
  "Original": "SIP/2.0 200 OK\r\nContent-Length: 0\r\n\r\n",
  "Discarded": 0,
  "NoStartLine": 0,
  "StartLine": 1,
  "NoHeaders": 0,
  "Header": 1,
  "NoBody": 0,
  "Body": 1,
  "Complete": 1,
  "Discards": [],
  "Starts": [
    "SIP/2.0 200 OK\r\n"
  ],
  "Headers": [
    "Content-Length: 0\r\n"
  ],
  "Bodies": [
    ""
  ],
  "Messages": [
    "SIP/2.0 200 OK\r\nContent-Length: 0\r\n\r\n"
  ]
}